ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsStateStore.java

```diff
@@ -73,28 +73,22 @@ KsqlConfig getKsqlConfig() {
 
   <T> T store(final QueryableStoreType<T> queryableStoreType, final int partition) {
     try {
-      final StoreQueryParameters<T> parameters = StoreQueryParameters.fromNameAndType(
-          stateStoreName, queryableStoreType).withPartition(partition);
-      if (ksqlConfig.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)
```

Comment:
Do we still need to check the value of `ksqlConfig.getBoolean(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED`?

Reply:
yes

Reply:
no we just check if we are using a Named topology related object

Reply:
What would have happened previously if someone had set `KSQL_SHARED_RUNTIME_ENABLED` to false?  (It seems that it would have prevented using a NamedTopology even if one were configured.

There is a logic change here.  Is it one we want?  (Sorry to ask twice; just trying to be sure about things.)

Reply:
That was the issue. We want this method to work with what ever store it is give regardless of `KSQL_SHARED_RUNTIME_ENABLED`